docs/books/computer-games/computer-battle-games/the_vital_message/the_vital_message.py

```diff
@@ -1 +1,47 @@
 
+# Import 'random' so we can generate random numbers.
+# The 'string' module gives us access to string-related utilities
+#The 'time' module helps us to wait for some time
+
+
+import random
+import string
+import time
+print("VITAL MESSAGE")
+
+#Variable to store the difficulty i.e length of string
+D=0
```

Comment:
BASIC requires single letter variable names - we can do a lot better with Python :-)